src/tests/Common/Coreclr.TestWrapper/CoreclrTestWrapperLib.cs

```diff
@@ -220,6 +219,7 @@ static bool CollectCrashDump(Process process, string path)
             {
                 createdump.StartInfo.FileName = "sudo";
                 createdump.StartInfo.Arguments = $"{createdumpPath} " + arguments;
+                createdump.StartInfo.EnvironmentVariables.Add("COMPlus_DbgEnableElfDumpOnMacOS", "1");
```

Comment:
Do we need this if you are already setting this variable from testenvironment.proj?

Reply:
I wasn't sure if when createdump is run here that the environment it inherits would contain the env vars from testenvironment.proj.  I guess I was just being cautious.